tests/foreman/ui/test_oscappolicy.py

```diff
@@ -125,7 +125,7 @@ def test_positive_check_dashboard(
         )
         policy_details = session.oscappolicy.details(name)
         assert policy_details['HostsBreakdownStatus']['total_count'] == 1
-        assert policy_details['HostBreakdownChart']['hosts_breakdown'] == '100%'
+        assert policy_details['HostBreakdownChart']['hosts_breakdown'] == '100%Not audited'
```

Comment:
What's up with that "Not audited"?
Seems similar to change we had recently in another PR: https://github.com/SatelliteQE/robottelo/pull/7845#discussion_r443466606

For me, the key question is: is this something we explicitly want to check, or is it something that just happened to appear and we want to see test still passing?

If that's second option, I would recommend checking Airgun if it's possible to put this new data in it's own widget. If it's not feasible, I would just drop this part of string before assertion.

Reply:
@mirekdlugosz I want to check this explicitly.